orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/Message.kt

```diff
@@ -197,17 +195,42 @@ data class CompleteStage(
   override val executionType: Class<out Execution<*>>,
   override val executionId: String,
   override val application: String,
-  override val stageId: String,
-  val status: ExecutionStatus
+  override val stageId: String
 ) : Message(), StageLevel {
-  constructor(source: ExecutionLevel, stageId: String, status: ExecutionStatus) :
-    this(source.executionType, source.executionId, source.application, stageId, status)
+  constructor(source: ExecutionLevel, stageId: String) :
+    this(source.executionType, source.executionId, source.application, stageId)
+
+  constructor(source: StageLevel) :
+    this(source.executionType, source.executionId, source.application, source.stageId)
 
-  constructor(source: StageLevel, status: ExecutionStatus) :
-    this(source, source.stageId, status)
+  constructor(source: Stage<*>) :
+    this(source.getExecution().javaClass, source.getExecution().getId(), source.getExecution().getApplication(), source.getId())
+}
 
-  constructor(source: Stage<*>, status: ExecutionStatus) :
-    this(source.getExecution().javaClass, source.getExecution().getId(), source.getExecution().getApplication(), source.getId(), status)
+data class SkipStage(
```

Comment:
Because the `CompleteStageHandler` will not have the context required to know the stage was skipped, I created a new message type.